docs/README.md

```diff
@@ -335,29 +304,16 @@ And finally kudos for all the people that believed in, tried, validated and even
 
 -   Feel free to send small pull requests. Please discuss new features or big changes in a GitHub issue first.
 -   Use `npm test` to run the basic test suite, `npm run coverage` for the test suite with coverage and `npm run test:performance` for the performance tests.
--   Please note that if you want to backport a feature / fix to MobX 4 a second PR needs to be opened to the mobx4-master branch.
 
-# MobX 4 vs MobX 5
+# Limitations without Proxy support
```

Comment:
I think this can be moved to a separate page, is this is not as common anymore as it was. Even RN supports proxies nowadays.

Reply:
I've now reshuffled matters: removed it from the home page, introduced a new page about these limitations, the pitfalls page adjusted to still have entries but to refer to the new page for details.